src/main/scala/io/buoyant/linkerd/RateLimiterConfig.scala

```diff
@@ -0,0 +1,26 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.finagle.{Filter, Stack}
+import io.buoyant.linkerd.protocol.HttpRequestAuthorizerConfig
+
+class RateLimiterConfig(limit: Int, intervalSeconds: Int = 1) extends HttpRequestAuthorizerConfig{
+  require(limit > 0, "The request limit must be greater than 0 requests.")
+  require(intervalSeconds > 0, "The interval must be greater than 0 seconds.")
+
+  @JsonIgnore
+  override def role: Stack.Role = Stack.Role("HttpRateLimiter")
+
+  @JsonIgnore
+  override def description: String = "HTTP Rate Limiter"
+
+  @JsonIgnore
+  override def parameters: Seq[Stack.Param[_]] = Seq()
+
+  @JsonIgnore
+  override def mk(params: Stack.Params): Filter[Request, Response, Request, Response] = {
+    new RateLimiter(limit, DefaultTimer, intervalSeconds)
```

Comment:
You can convert this to a duration if you `import com.twitter.conversions.time._` and do `intervalSeconds.seconds`